src/main/java/io/debezium/connector/yugabytedb/consistent/Merger.java

```diff
@@ -40,6 +64,16 @@ public BigInteger streamSafeTime() {
         return Collections.min(tabletSafeTime.values());
     }
 
+    /**
+     * Set the safetime for the tablet to the given value.
+     * @param tabletId the tablet UUID for which the safetime needs to be set
+     * @param safeTime the safetime to be set
+     */
+    public void setTabletSafeTime(String tabletId, BigInteger safeTime) {
```

Comment:
I am assuming `setTabletSafeTime` will be called only from `add`. So maybe a duplicate of assert in `add`.

An assert that safeTime can only move forward (or stay in place) will be good to have hear.